index.bs

```diff
@@ -552,18 +552,36 @@ The header's ABNF is:
 The 'Sec-CH-UA-Form-Factor' Header Field {#sec-ch-ua-form-factor}
 ------------------------------
 
-The <dfn http-header>`Sec-CH-UA-Form-Factor`</dfn> request header field gives a server information
-about the [=user agent=]'s [=form-factor=]. It is a [=Structured Header=] whose value MUST be a
-[=structured header/string=]. Its value SHOULD match one of the following common form-factor values:
-"Automotive", "Mobile", "Tablet", "TV", "VR", "XR", "Unknown" or the empty string.
-[[!RFC8941]].
+The <dfn http-header>`Sec-CH-UA-Form-Factor`</dfn> request header field gives a
+server information about the [=user agent=]'s [=form-factor=]. It is a
+[=Structured Header=] whose value MUST be a [=structured header/list=]
+[[!RFC8941]]. Its values SHOULD describe the form-factor of the device using
+one or more of the following common form-factor values: "Desktop",
+"Automotive", "Mobile", "Tablet", "TV", "VR", or "XR". Order of the values
+in the list is not significant.
+
+<div class="note" heading="">
```

Comment:
This `heading=""` seems to be required in order to get the green "NOTE:"?

Reply:
That's weird! You can also just do:

`Note: ....` and it'll generate the markdown for you.

https://speced.github.io/bikeshed/#notes-etc does reference the heading attribute. But if what you have works, I'm not particularly worried about it.

Reply:
Should this be a note at all? It seems like important normative content

Reply:
My understanding was that the language is too vague to be normative, and making it more precise would be counter-productive. But I definitely can't afford an editor's hat, so I'll defer to those with better headware.

Reply:
I don't have strong opinions one way or the other here, and it's easy to swap between the two.